packages/lesswrong/lib/collections/conversations/schema.ts

```diff
@@ -77,6 +79,7 @@ const schema: SchemaType<DbConversation> = {
     canRead: ['guests'],
     canCreate: ['members'],
     canUpdate: ['members'],
+    ...schemaDefaultValue([]), // Note: this sets onUpdate and onCreate. onUpdate is overridden below
```

Comment:
Required to make `useInitiateConversation` work correctly. The bug here was that this would be null when the conversation was created (it was only set to `[]` after a message was sent), which excluded it from the conversations fetched by `"userGroupUntitledConversations"` in `useInitiateConversation` so it wouldn't navigate to a newly created conversation.

This wasn't a problem in the old version because we used `useCreate` directly